app/assets/stylesheets/admin/loans/timeline/_table.scss

```diff
@@ -1,11 +1,34 @@
 @import "../../colors";
 
 .timeline-table {
-  table {
+  .date-col {
+    min-width: 8rem;
+  }
+
+  // Specificity used to override Bootstrap defaults
+  .table thead:first-child tr {
+    th {
+      border: 1px solid $table-border-color;
+      border-bottom-width: 2px;
+    }
+  }
+
+  td, th {
     background: $white;
   }
 
-  .date-col {
-    min-width: 8rem;
+  td {
+    border: 1px solid $table-border-color;
+
+    &.spacer {
+      background: $background-gray;
+      border: none;
+    }
+  }
+
+  .fa-cog {
+    color: $gray3;
+    font-size: 1.2em;
```

Comment:
I went ahead and used the new lightest gray we are using and reduced the text size.

Just a note, the lighter grays, including the one in mockup of #c0c0c0/ rgb(192, 192, 192) are not accessible on a white background. See http://webaim.org/resources/contrastchecker.